playwright/capture.spec.ts

```diff
@@ -52,6 +52,7 @@ test.describe('event capture', () => {
 
         // Pageview will be sent immediately
         await pollUntilEventCaptured(page, '$pageview')
+        await pollUntilCondition(page, () => captureRequests.length > 0)
```

Comment:
this was unrelated, but this failed on a clean branch on my local machine until I added this poll here